examples/csi-app.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   serviceAccountName: default
   containers:
     - name: my-frontend
-      image: quay.io/quay/busybox
+      image: registry.access.redhat.com/ubi8/ubi
```

Comment:
not that this is "bad", but why did you feel the need to change this?

Reply:
nope that is sufficient justification for me ... let's continue with ubi image .... that is "more strategic" from a RH perspective anyway